client/src/common/styledComponents/List.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'vue-styled-components';
+
+export const List = styled.div`
```

Comment:
Minor: Is it possible to use <ul> tag here instead of div?

Reply:
Actually I didn't find the way how to do it in Vuejs, if you know how to do it please let me know.